### DISCUSSION TITLE
Mic

### DISCUSSION THREAD

**Discussion by KurdiKurdistan** [sebanc/brunch#781] [Q&A]
Hello people i already installed Chrome os  on my Dell laptop everything ok and great

but i have mic issue my mic stopped working when i installed chrome os from brunch framework

thanks

**Comment by kerta1n** [comment]
You didn't read the docs, but you should follow this [tutorial](https://www.youtube.com/watch?v=db4Xo_xPTR4). IMPORTANT: around 2:24 he will add enable_updates, you should also apply internal_mic_fix so that it looks like "options=option1,option2,..." (without spaces). For example: booting with "options=enable_updates,advanced_als" will activate both options.
